packages/server/src/contacts/__tests__/contacts.controller.spec.ts

```diff
@@ -95,7 +95,7 @@ describe('ContactsController', () => {
           name: 'Joe',
         } as Contact);
       } catch (err) {
-        expect(err.message.message).toEqual('Contact address must have ETH format');
+        expect(err.message.message).toEqual('This method only supports 0x-prefixed hex strings but input was: undefined');
```

Comment:
why is this changed? i think we should have ETH formatted addresses even in the tests, there is a new test for this

Reply:
Because of the update on eth util update. It will throw that error in case it is not a good format